src/garage/tf/models/gaussian_lstm_model.py

```diff
@@ -0,0 +1,234 @@
+"""GaussianLSTMModel."""
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.core.lstm import lstm
+from garage.tf.core.parameter import parameter
+from garage.tf.distributions import DiagonalGaussian
+from garage.tf.models import Model
+
+
+class GaussianLSTMModel(Model):
+    """
+    GaussianLSTMModel.
+
+    Args:
+        output_dim (int): Output dimension of the model.
+        hidden_dim (int): Hidden dimension for LSTM cell for mean.
+        name (str): Model name, also the variable scope.
+        hidden_nonlinearity (callable): Activation function for intermediate
+            dense layer(s). It should return a tf.Tensor. Set it to
+            None to maintain a linear activation.
+        hidden_w_init (callable): Initializer function for the weight
+            of intermediate dense layer(s). The function should return a
+            tf.Tensor.
+        hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s). The function should return a
+            tf.Tensor.
+        recurrent_nonlinearity (callable): Activation function for recurrent
+            layers. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        recurrent_w_init (callable): Initializer function for the weight
```

Comment:
Hmm I believe it is not callable right? As far as I saw it is an initializer instance

Reply:
tf initializers are callables which return tensors